### PR TITLE
[EdgeTPU] Create new CfgEditorPanel for Edgetpu

### PR DESCRIPTION
**Pull request by hohee-hee**
- Create ICfgEditorPanel.ts : interface for EdgetpuCfgEditorPanel and CfgEditorPanel
- Create EdgetpuCfgEditorPanel.ts : Cfg Editor Panel for edgetpu cfg editor
- Add register in extension.ts
- Map a new view type with a new file extension - .edgetpufcg

ONE-vscode-DCO-1.0-Signed-off-by: hohee-hee <khappy517@gmail.com>

### CODE REVIEW

**Review by rhqjatn2398 — APPROVED** [review]
LGTM

File: src/CfgEditor/EdgetpuCfgEditorPanel.ts
Comment (Hyeon-Uk) [inline src/CfgEditor/EdgetpuCfgEditorPanel.ts:1]: I think, only initWebview function is affected by the file extension name.

So I think it's better to write a code that injects different src for each extension into the existing file without separating it into a file.

Comment (hohee-hee) [reply src/CfgEditor/EdgetpuCfgEditorPanel.ts:1]: The main modification of EdgetpuCfgEditor.ts was not only `initWebview` but `viewType`.

Unfortunately, viewType is a key value when displaying editor view. This cannot be mapped in the 'CfgEditorPanel.ts' depending on file extension name. So duplication of almost code is inevitable😂

Comment (rhqjatn2398) [reply src/CfgEditor/EdgetpuCfgEditorPanel.ts:1]: I agree.

Comment (Hyeon-Uk) [reply src/CfgEditor/EdgetpuCfgEditorPanel.ts:1]: It's very unfortunate :(

**Pull request merged**